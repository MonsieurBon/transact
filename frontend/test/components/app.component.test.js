import { fixture } from '@open-wc/testing-helpers';
import App from '../../src/components/app.component.js';

describe('App', () => {
  beforeAll(() => {
    window.customElements.define('t-app', App);
  });

  it('should render', async () => {
    const el = await fixture('<t-app></t-app>');
    expect(el.shadowRoot.innerHTML).toMatchSnapshot();
  });
});
