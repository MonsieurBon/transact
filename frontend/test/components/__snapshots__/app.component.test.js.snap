// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
"
<style>
  :host {
    display: flex;
    contain: content;
    height: 100%;
    width: 100%;
    flex-direction: column;
  }
  
  .header {
    background: black;
    height: 100px;
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    align-items: center;
    padding: 0 20px;
  }
  
  .header h1 {
    color: white;
  }
  
  .main {
    flex-grow: 1;
    display: flex;
    flex-direction: row;
  }
  
  .side {
    background: lightgray;
    width: 200px;
    border-right: 2px solid gray;
    box-shadow: 2px 3px 4px 1px darkgray;
  }
  
  .content {
    flex-grow: 1;
  }
</style>

<div class=\\"header\\">
  <h1>Transact</h1>
  <user-info></user-info>
</div>
<div class=\\"main\\">
  <div class=\\"side\\"></div>
  <div class=\\"content\\"></div>
</div>
"
`;
